main.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import glob
+import re
+import os
+# Ścieżka do folderu, w którym znajdują się pliki
+folder_path = os.getcwd() + '\data'
+
+def clean_text(text):
+    # Usunięcie znaków interpunkcyjnych
+    cleaned_text = re.sub(r"[^\w\s]", "", re.sub(r"[\t\n]", "", text))
+
+    return cleaned_text
+
+def get_data(folder_path):
+    # Pobieranie plików z rozszerzeniami .xlsx, .csv i .txt
+    file_extension = ['xlsx', 'csv', 'txt']

```

Comment:
Best to keep it as global variable at the top of a script so It cacn be easily changed